trello.py

```
@@ -1,52 +1,99 @@
+from datetime import datetime
 
-CREDS = {
-    "key": TRELLO_KEY,
-    "token": TRELLO_TOKEN
-}
+DATE_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
 
-trelloLists = get(f"{TRELLO_URL}/boards/{TRELLO_BOARD}/lists", params=CREDS).json()
-listIdToName = {list['id']: list['name'] for list in trelloLists}
-listNameToId = {list['name']: list['id'] for list in trelloLists}
+class TrelloConfig:
+    def __init__(self, key, token, url, board_id):
+        self._credentials = {
+            "key": key,
+            "token": token
+        }
+        self._url = url
+        self._board_id = board_id
 
-assert(set(listNameToId.keys()) == set([NOT_STARTED, COMPLETED]))
+    @property
+    def credentials(self):
+        return self._credentials
 
-def get_items():
-    cards = get(f"{TRELLO_URL}/boards/{TRELLO_BOARD}/cards", params=CREDS).json()
-    return [parseTrelloCard(card) for card in cards]
+    @property
+    def url(self):
+        return self._url
 
+    @property
+    def board_id(self):
+        return self._board_id
 
-def get_item(id):
-    response = get(f"{TRELLO_URL}/cards/{id}", params=CREDS)
-    if response.status_code != 200:
-        return None
-    return parseTrelloCard(response.json())
 
+class Trello:
+    def __init__(self, config, api_client):
+        self._url = config.url
+        self._credentials = config.credentials
+        self._board_id = config.board_id
+        self._api_client = api_client
 
-def add_item(title):
-    post(f"{TRELLO_URL}/cards", params={
-        "name": title,
-        "idList": listNameToId[NOT_STARTED],
-        **CREDS
-    })
+        trelloLists = self._api_client.get(f"{self._url}/boards/{self._board_id}/lists", self._credentials).json()
+        self._listIdToName = {list['id']: list['name'] for list in trelloLists}
+        self._listNameToId = {list['name']: list['id'] for list in trelloLists}
+        assert(set(self._listNameToId.keys()) == set([NOT_STARTED, IN_PROGRESS, COMPLETED]))
```

You should raise an exception with a more helpful message rather than using `assert` in actual app code. Yes, a user/developer can look at this line and then look at the definition of the 3 strings to see what lists are required. But they shouldn't have to. 

Better yet would be to document it in the README.